products-dashboard/definition.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Also, please check this file because it contains default information. Change them with data related to the project you are creating